Homework3/Task2Stack/ReverseExtension.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task2Stack
+{
+    static class ReverseExtension
+    {
+        public static Stack<T> Reverse<T>(this IStack<T> stack) // Why does the method name need to include <T> here, but not in IStack
+        {
+            return new Stack<T>(default);
```

Comment:
The method should reverse existing Stack. 

Pseudo code:

Create a new empty stack
Pop each element from existing stack and push it into the new one

Reply:
How do I use an existing stack if I'm not allowed to pass it as a parameter into this method?